rootfs/standard/usr/bin/service_scripts/pre_lnbits.sh

```diff
@@ -1,23 +1,38 @@
 #!/bin/bash
 
 set -x
+source /usr/share/mynode/mynode_app_versions.sh
 
-# Copy config file
-if [ ! -f /mnt/hdd/mynode/lnbits/.env ]; then
-    cp /usr/share/mynode/lnbits.env /mnt/hdd/mynode/lnbits/.env
+# Delete old myNode .env file created before 0.12.x
+if [ -f /usr/share/mynode/lnbits.env ]; then
+    rm -f /usr/share/mynode/lnbits.env
```

Comment:
I'd rather not get rid of this file since I can control it a bit better and reference the config. Can it just be updated with the latest version from the lnbits repo? If I do have to put in default settings, this file (/usr/share/mynode/lnbits/env) is a nice way to do it.

Reply:
Will fix the way you suggested